<launch>
    <node pkg="joy" type="joy_node" name="joy_node">
        <param name="dev" value="/dev/input/js0"/>
        <param name="deadzone" value="0.1"/>
    </node>

    <!-- ROV Joy Controller -->
    <node pkg="v2_control" type="joy-control" name="joy_control" output="screen">
        <!-- Joystick mapping -->
        <param name="axis_surge" value="1"/>
        <param name="axis_sway" value="0"/>
        <param name="axis_heave" value="4"/>
        <param name="axis_yaw" value="3"/>
        <!-- Scale factors -->
        <param name="surge_scale" value="75.0"/>
        <param name="sway_scale" value="50.0"/>
        <param name="heave_scale" value="5.0"/>
        <param name="yaw_scale" value="5.0"/>
    </node>
</launch>